notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2359 @@
+      "text/plain": [
+       "<Figure size 1000x1000 with 4 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Generate a visualization of the indirect missing data percentages for each feature\n",
+    "GLAD.plot_indirect_missing(indirect_missing)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I will **delete any features with 60% or more missing data**, regardless of the type of missing data. "
```

Comment:
Clarify that deleted features only from "not_required" missing type

Reply:
Every feature or column with more than 60% indirect missing data is removed. not only for the "not required" type.